roles/ceph-rgw/tasks/multisite/secondary.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: check connectivity to other cluster's rgw endpoints
```

Comment:
Because we're not sure that curl command is available on the nodes.
```yaml
- name: check connectivity to other cluster's rgw endpoints
  uri:
    url: '{{ item }}'
  with_items: "{{ rgw_primary_endpoints }}"
  when: rgw_primary_endpoints is defined
```